docs/blog-articles/02-ideal-bpm-brazilian-zouk-en.md

```
@@ -0,0 +1,75 @@
+# ARTICLE 02 — ENGLISH
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)